README.md

```
@@ -1 +1,60 @@
+
+3. **Create .env file**
+
+Create .env file at root of project, provide required variables listed in .env.template
+
+3. **Run the Development Server**
+
+    ```bash
+    yarn start:dev
+    ```
+
+    The app will typically be available at http://localhost:3000, but check the terminal output for the exact URL.
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)